frontend/src/metabase/querying/filters/types.ts

```diff
@@ -74,11 +75,12 @@ export type DatePickerValue =
 
 export type DatePickerValueType = DatePickerValue["type"];
 
-export type RelativeIntervalDirection = "last" | "current" | "next";
```

Comment:
Nit: I think `"past" | "current" | "future"` would be more accurate (since `last` and `next` can be overloaded terms in the context of time).